storybook/.storybook/preview.ts

```diff
@@ -1,34 +1,45 @@
-import { Preview } from "@storybook/web-components";
+import { GlobalTypes, DecoratorFunction, Parameters } from "@storybook/types";
 import { i18nDecorator } from "./i18n.decorator";
 
-const preview: Preview = {
-  globalTypes: {
-    locale: {
-      description: "Internationalization locale",
-      defaultValue: "nl",
-      toolbar: {
-        icon: "globe",
-        items: [
-          { value: "nl", left: "🇳🇱", title: "Nederlands" },
-          { value: "en", left: "🇬🇧", title: "English" },
-        ],
-      },
+export const globalTypes: GlobalTypes = {
+  locale: {
+    description: "Internationalization locale",
+    defaultValue: "nl",
+    toolbar: {
+      icon: "globe",
+      items: [
+        { value: "nl", left: "🇳🇱", title: "Nederlands" },
+        { value: "en", left: "🇬🇧", title: "English" },
+      ],
     },
   },
-  decorators: [(story, context) => i18nDecorator(story, context)],
-  parameters: {
-    html: {
-      root: "#dt-i18n-decorator-container", // default: #root
-      removeEmptyComments: true, // default: false
-      removeComments: /\?.*/, // default: false
-    },
-    options: {
-      storySort: {
-        method: "alphabetically",
-        order: ["HTML|CSS", "Core"],
-      },
+};
+
+export const decorators: DecoratorFunction[] = [(story, context) => i18nDecorator(story, context)];
+
+export const parameters: Parameters = {
+  html: {
+    root: "#dt-i18n-decorator-container", // default: #root
+    removeEmptyComments: true, // default: false
+    removeComments: /\?.*/, // default: false
+  },
+  options: {
+    storySort: {
+      method: "alphabetically",
+      order: ["HTML|CSS", "Core"],
     },
   },
 };
 
-export default preview;
+import "@iframe-resizer/child";
+import sprite from "dso-toolkit/dist/dso-icons.svg?raw";
```

Comment:
Todo: Type fix

Reply:
De Storybook bundler gaat hier precies mee om zoals we willen, TypeScript klaagt echter. Ik denk dat de simpele oplossing (`@ts-expect-error`) beter is dan een `.d.ts` file inchecken en de module te declareren.